apps/omg_watcher/test/omg_watcher/integration/block_getter_test.exs

```diff
@@ -215,4 +215,34 @@ defmodule OMG.Watcher.Integration.BlockGetterTest do
     # we should still be able to challenge this "unchallenged exit" - just smoke testing the endpoint, details elsewhere
     WatcherHelper.get_exit_challenge(exit_blknum, 0, 0)
   end
+
+  @tag fixtures: [:in_beam_watcher, :mix_based_child_chain, :test_server, :stable_alice, :stable_alice_deposits]
+  test "operator claim more fees than collected", %{
```

Comment:
let's talk about this test - @achiurizo might want to see this in cabbage, instead of 
```
:in_beam_watcher, :mix_based_child_chain
```

Reply:
my 2 cents on this:

Does cabbage support invalidly behaving child chain servers in current `master`? I assume it doesn't yet, since [this test](https://github.com/omisego/elixir-omg/pull/1312/files#diff-cd2d99842e0db23a5cf13c37248c98a3R148) is still here.

When it will, moving all those tests over to cabbage can be done in bulk. Since this PR isn't adding the first test like this, I think it's acceptable and pragmatic to add such a test now and refactor later.

Reply:
Cabbage can support whatever is needed. If it’s not built, it needs to be built.

No one is specifically assigned to port all the tests (and, by the amount of tests, this is truly a horrible task to give someone).
So to me, the pragmatic approach is: don’t add anymore tests that need to be rewritten or refactored, start doing the changes now.

Reply:
My 2 cents, from the prev experience with cabbage, it’s great, but very immature, lets just recall #1293 or it does not provide tests isolation. We will overcome all these issues at some point, however porting this case now it’s way bigger than the task itself.

Anyway, it’s about priorities. 
CC @T-Dnzt

Reply:
I'll return with 6 cents:

> Does cabbage support invalidly behaving child chain servers in current master? I assume it doesn't yet, since this test is still here.
> Since this PR isn't adding the first test like this, I think it's acceptable and pragmatic to add such a test now and refactor later.
> We will overcome all these issues at some point, however porting this case now it’s way bigger than the task itself. Anyway, it’s about priorities.

The points raised here make sense and I can see that adopting a new framework or technology could make this transition difficult for some. The team and myself are happy to assist you all to help us all _collectively_ move to a better testing setup that's going to benefit all the engineers and contributors of this codebase.

In order for us to move forward to make it better for everyone, we all need to contribute. If we casually push aside these improvements because "it wasn't built" or there was "prior art so we can add on it" or that its "not priorities", these improvements will fall short.
> So to me, the pragmatic approach is: don’t add anymore tests that need to be rewritten or refactored, start doing the changes now.

I think this is quite a fair stance. I think the history of the code base can show that the "punt now, refactor later" approach has not done well(recall docker/mix issues and documents being out of date, difficulty getting distillery introduced, testing, etc ...) and to continue that behavior here is only going to add to that same pile.

---

As mentioned before, happy to offer time to pair to move this forward with cabbage. I think it's safe to say we all want to improve the code base and that it should be our collective priority to improve this now, and enjoy the benefits later.

Reply:
Note: We agreed to postpone cabbage test due to tight timeframes